packages/swapper/src/swappers/cow/getCowSwapMinMax/getCowSwapMinMax.ts

```diff
@@ -0,0 +1,34 @@
+import { Asset } from '@shapeshiftoss/types'
+
+import { MinMaxOutput, SwapError, SwapErrorTypes } from '../../../api'
+import { bn, bnOrZero } from '../../utils/bignumber'
+import { CowSwapperDeps } from '../CowSwapper'
+import { MAX_COWSWAP_TRADE } from '../utils/constants'
+import { getUsdRate } from '../utils/helpers/helpers'
+
+export const getCowSwapMinMax = async (
+  deps: CowSwapperDeps,
+  sellAsset: Asset,
+  buyAsset: Asset
+): Promise<MinMaxOutput> => {
+  try {
+    if (
+      !sellAsset.assetId.startsWith('eip155:1/erc20') ||
+      !buyAsset.assetId.startsWith('eip155:1/erc20')
```

Comment:
this is not ideal coupling this implementation to mainnet. cowswap supports gnosis too and we should be considering this when implementing